tests/signUpValidation.spec.js

```diff
@@ -0,0 +1,157 @@
+import {test, expect} from "@playwright/test";
+
+test.describe("Validation", ()=>{
+    test.beforeEach(async ({page})=>{
+        await page.goto("https://qauto.forstudy.space/")
+        await page.click('text="Sign up"')
+    })
+
+    test("Name", async ({page})=>{
+        const inputNameString ="Serhii"
+
+        const popup = page.locator('app-signup-modal')
+        const nameInput = popup.locator('#signupName')
+        const nameErrorMessage =  popup.locator('p', {hasText: 'Name required'})
+        const nameErrorMessage2 =  popup.locator('p', {hasText: 'Name is invalid'})
+        const nameErrorMessage3 =  popup.locator('p', {hasText: 'Name has to be from 2 to 20 characters long'})
+
+        await nameInput.focus()
+        await nameInput.blur()
+        await expect(nameErrorMessage).toBeVisible()
+        await nameInput.fill("Сергій")
+        await nameInput.blur()
+        await expect(nameErrorMessage2).toBeVisible()
+        await nameInput.fill("asdkleqomroqlfkrpsmrs")
+        await nameInput.blur()
+        await expect(nameErrorMessage3).toBeVisible()
+
+        await nameInput.fill("a")
+        await nameInput.blur()
+        await expect(nameErrorMessage3).toBeVisible()
+
+        await nameInput.fill(inputNameString)
+
+        await expect(nameInput).toHaveValue(inputNameString)
+    })
+
+    test("LastName", async ({page})=>{
+        const inputLastNameString ="Opalinskyi"
+
+        const popup = page.locator('app-signup-modal')
+        const lastNameInput = popup.locator('#signupLastName')
+        const lastNameErrorMessage =  popup.locator('p', {hasText: 'Last name required'})
+        const lastNameErrorMessage2 =  popup.locator('p', {hasText: 'Last name is invalid'})
+        const lastNameErrorMessage3 =  popup.locator('p', {hasText: 'Last name has to be from 2 to 20 characters long'})
```

Comment:
Even though I don't really like this approach when you are trying to find error element by text I have to admit it might make sense.

But I'd prefer to search by CSS selector and later use 
https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-text

in this case you would have one variable for locator instead of 3